src/components/AppSidebar/AppSidebarTabs.vue

```diff
@@ -213,15 +213,20 @@ export default {
 		 * Manually update the sidebar tabs according to $slots.default
 		 */
 		updateTabs() {
+			if (!this.$slots.default) {
+				this.tabs = []
```

Comment:
A `return` is missing here, otherwise the rest of the function this runs and throws a `TypeError: this.$slots.default is undefined` as it currently happens on page load for the Netlify doc.